Engine/Src/Ancona/Core2D/Systems/Drawable/Drawable.cpp

```diff
@@ -54,3 +57,32 @@ Drawable * Drawable::FindDrawable(const std::string &key)
     return nullptr;
 }
 
+void Drawable::FlipX()
```

Comment:
You should remove FlipX and FlipY. If there is an idempotent way to implement an API (like SetXDirection), that should be the only API provided. Idempotent APIs are generally better behaved.